src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,71 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Calculator {
+
+    private final String DEFAULT_DELIMITER = "[,:]";
+    private final int LAST_PREFIX_INDEX = 4;
+    private final int CUSTOM_DELIMITER_INDEX = 2;
+
+    public int sumAll(String input) {
+        if (input.isBlank()) {
+            return 0;
+        }
+
+        String delimiter = DEFAULT_DELIMITER;
+        if (input.startsWith("//")) {
+            delimiter = updateCustomDelimiter(input);
+            input = removeCustomDelimiterPrefix(input);
+        }
+
+        return getSum(makeStringToIntArray(delimiter, input));
+    }
+
+    private String updateCustomDelimiter(String input) {
+        return String.valueOf(DEFAULT_DELIMITER.charAt(0)) + String.valueOf(input.charAt(CUSTOM_DELIMITER_INDEX)) + DEFAULT_DELIMITER.substring(1);
+    }
+
+    private String removeCustomDelimiterPrefix(String input) {
+        return input.substring(LAST_PREFIX_INDEX + 1);
+    }
+
+
+    private List<Integer> makeStringToIntArray(String delimiter, String input) {
+        String[] stringNumbers = input.split(delimiter);
+
+        List<Integer> numbers = new ArrayList<>();
+        for (String stringNumber : stringNumbers) {
+            numbers.add(checkAndTransformToInt(stringNumber));
+        }
+        return numbers;
+    }
+
+    private Integer checkAndTransformToInt(String stringNumber) {
```

Comment:
```
Q2) 여러 private 메서드들을 만들었는데, 매개변수나 반환 값으로 원시값 int로 선언하는게 좋을지 Integer 래퍼 클래스로 선언하는게 좋은지 궁금합니다.
```

```suggestion
    private int checkAndTransformToInt(String stringNumber) {
```

nullable 하지 않으면 굳이 래퍼 클래스를 사용할 이유는 없습니다.

Reply:
감사합니다. nullable 한 것과 아닌것은 앞으로도 명확히 구분해서 사용해야겠습니다!!

Reply:
@dhmin5693 혹시 Resolve conversation 은 제가 클릭해서 resolve로 바꿔야 하는걸까요??

Reply:
@jin-mook 안바꾸셔도 큰 상관은 없습니다. 저는 resolve 상태로 바뀌었다면 내용을 확인하셨다는걸 더 잘 인지할 수 있어서 선호하는데, 다른 리뷰어 분들은 어떻게 받아들일지는 모르겠네요.

Reply:
넵 그러면 피드백 내용이 한번에 보이는게 좋은것 같아서 따로 바꾸지 않겠습니다